core/src/main/resources/lib/layout/html.jelly

```diff
@@ -0,0 +1,177 @@
+<!--
```

Comment:
Yes. This is a full-blown jenkins page, without the decoration. Not ideal, but I'm just reusing the functionality from security config and such in the setup wizard GUI. this allows any plugin to continue to work to contribute its own configuration, etc.